src/routes/tokenGovernance.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Tabs } from 'antd';
+
+import Web3SubmitWarning from '../components/web3SubmitWarning';
+
+import { Button, Form, Input, Row, Col } from 'antd';
+import TextArea from 'antd/lib/input/TextArea';
+import { keccak256 } from 'ethereumjs-util';
+import Web3 from '../stores/web3/ts_workaround';
+import { tokenGovernanceContractStore } from '../stores/tokenGovernanceContract';
+
+const { Fragment } = React;
+const { TabPane } = Tabs;
+
+export default class TokenGovernance extends React.Component {
+  public state: { title: string; description: string };
+  public sendProposal: any;
+
+  constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      title: '',
+      description: '',
+    };
+  }
+
+  public handleClick = e => {
+    this.setState({
+      current: e.key,
+    });
+  };
+
+  public render() {
+    function callback(key) {
+      console.log(key);
```

Comment:
debug output? Remove?

Reply:
this `function callback(key)` is from the ant.design framework for the tab element. But i do not need the `console.log`

Reply:
I think `onChange` is optional for `Tabs` and can be omitted if not used